netci.groovy

```diff
@@ -1442,6 +1442,11 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
                         buildCommands += "set __TestIntermediateDir=int&&build.cmd ${lowerConfiguration} ${arch} ${buildOpts}"
                     }
 
+                    // If it is a release build for windows, ensure PGO is used, else fail the build
```

Comment:
Looks like this change will get picked up for x86 Legacy Backend and x86 Ryujit. I am assuming x64 and x86 Ryujit are the only interesting cases